k8s/namespaces/nfdiv/nfdiv-frontend/demo.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: nfdiv-frontend
+spec:
+  releaseName: nfdiv-frontend
+  chart:
+    git: git@github.com:hmcts/hmcts-charts
+    ref: master
+    path: stable/nfdiv-frontend
+  values:
+    nodejs:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      ingressClass: traefik-no-proxy
```

Comment:
do you need the replicas interpod and ingress host all set here?

Reply:
removed good point